src/yunohost/domain.py

```diff
@@ -204,8 +204,9 @@ def domain_remove(operation_logger, domain, force=False):
 
     for app in _installed_apps():
         settings = _get_app_settings(app)
+        label = app_info(app)["name"]
         if settings.get("domain") == domain:
-            apps_on_that_domain.append("%s (on https://%s%s)" % (app, domain, settings["path"]) if "path" in settings else app)
+            apps_on_that_domain.append("%s \"%s\" (on https://%s%s)" % (app, label, domain, settings["path"]) if "path" in settings else app)
 
     if apps_on_that_domain:
         raise YunohostError('domain_uninstall_app_first', apps=", ".join(apps_on_that_domain))
```

Comment:
```suggestion
        raise YunohostError('domain_uninstall_app_first', apps="\n    - " + "\n    - ".join(apps_on_that_domain))
```

Reply:
Can be nice to put the apps on separate lines ? (The join becomes ugly but better display :s )

Reply:
Ok :) (I hope it won't be a problem for the admin)